tests/dao_tests/test_questionnaire_response_dao.py

```diff
@@ -1172,6 +1179,7 @@ def test_insert_qr_three_times(self):
             questionnaireId=2,
             questionnaireVersion=1,
             participantId=1,
+            isCATI=False,
```

Comment:
Doesn't need to change at all (just nit-picking), but you could add a tuple to the data_generator's `_questionnaire_response` method (in the list of defaults at the top) with `('isCATI', False)` and not need to specify it here in these tests.

Reply:
Done